config/github.php

```diff
@@ -88,7 +88,7 @@
 
         'private' => [
             'appId'      => 'your-github-app-id',
-            'keyPath'    => 'your-private-key-path',
+            'keyPath'    => 'file://your-private-key-path',
```

Comment:
Requiring this to start with file is going to be a major breaking change that we can't make